EIPS/eip-5094.md

```diff
@@ -0,0 +1,95 @@
+---
+eip: 5094
+title: URI Format for Ethereum Network Switching
+description: A way of representing various network configurations as URLs.
+author: Luc van Kampen (@lucemans), Jakob Helgesson (@svemat01), Joshua Hendrix (@thejoshuahendrix)
+discussions-to: https://ethereum-magicians.org/t/5094-uri-format-for-ethereum-network-switching/9277
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-05-13
+requires: 681, 831
+---
+
+## Abstract
+
+This standard includes all needed information for adding a network to a wallet via URL, by including parameters such as `chainId`, `rpc_url`, `chain_name` and others, such that the network configuration is provided through the URL itself.
```

Comment:
The title uses `URI`, while the body uses `URL`. I'd suggest picking one for consistency.